src/collision.cpp

```diff
@@ -574,7 +583,8 @@ collisionMoveResult collisionMoveSimple(Environment *env, IGameDef *gamedef,
 				box.MinEdge += *pos_f;
 				box.MaxEdge += *pos_f;
 			}
-			if (std::fabs(cbox.MaxEdge.Y - box.MinEdge.Y) < 0.15f * BS) {
+			//if (std::fabs(cbox.MaxEdge.Y - box.MinEdge.Y) < 0.15f * BS) {
+			if (std::fabs(cbox.MaxEdge.Y - box.MinEdge.Y) < 0.05f) {					//TheTermos make it less stupid after step-up fix
```

Comment:
why is the `* BS` gone here?

Reply:
0.15 * BS translates to 15 world centimeters, an extreme value, that's the reason why touching_ground has been useless. 

0.05 should be enough. It relates to calculation precision rather than world dimensions, so it needn't be expressed in BS.